selfdrive/car/torque_data/params.yaml

```diff
@@ -22,6 +22,7 @@ HONDA HRV 2019: [2.0661212805710205, 0.7521343418694775, 0.17760375789242094]
 HONDA INSIGHT 2019: [1.5201671214069354, 0.5660229120683284, 0.25808042580281876]
 HONDA ODYSSEY 2018: [1.8774809275211801, 0.8394431662987996, 0.2096978613792822]
 HONDA PILOT 2017: [1.7262026201812795, 0.9470005614967523, 0.21351430733218763]
+HONDA PILOT 2023: [1.7262026201812795, 0.9470005614967523, 0.21351430733218763]
```

Comment:
You can use the substitute file if you're just copying the values

Reply:
Thanks! Added!